python/datasources/graphql_ahr.py

```diff
@@ -95,12 +95,12 @@
 
 RAW_TO_SHARE_ALL_AGES_MAP = {
     rate_col: f'{std_col.extract_prefix(rate_col)}_{std_col.PCT_SHARE_SUFFIX}'
-    for rate_col in AHR_MEASURES_TO_RATES_MAP_ALL_AGES.values()
+    for rate_col in RATE_TO_RAW_ALL_AGES_MAP.values()
```

Comment:
Can we use raw_col instead of rate_col for these? The iterated variable should match what's it's actually representing